ubuntu_package_buildinfo/cli.py

```diff
@@ -212,6 +231,13 @@ def get_buildinfo(
             print(f"**********ERROR: \tNo buildinfo found for {package_name} {package_architecture} version {package_version} in {package_series}. See {build_web_link} for more details. Source package {binary_build.source_package_name} version {binary_build.source_package_version}.")
         else:
             if download:
+                # These logs are parsed by internal Canonical tools
+                print(
+                    f"DEBUG: \tbinary build URL: {binary_build}"
+                    f"\nDEBUG: \tbuildinfo URL: {buildinfo_url}"
+                    f"\nDEBUG: \tbuildlog URL: {buildlog_url}"
+                    f"\nDEBUG: \tchangesfile URL: {changesfile_url}"
+                )
```

Comment:
The logging level flag is redundant as that is not used anywhere. These would be printed irrespective of logging level but should someone fix logging level flag later can convert these to DEBUG logs (provided that format of logging does not change)